<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetTransfers</class>
 <widget class="QWidget" name="WidgetTransfers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transfers</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Generic/Transfers.png</normaloff>:/Resource/Generic/Transfers.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterTransfers">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="treeWidgetTransfers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="headerVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Transfers</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>Downloads</string>
       </property>
       <property name="statusTip">
        <string notr="true">Downloads</string>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/Resource/Generic/Download.png</normaloff>:/Resource/Generic/Download.png</iconset>
       </property>
       <item>
        <property name="text">
         <string>Completed</string>
        </property>
        <property name="statusTip">
         <string notr="true">CompletedDownloads</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/Resource/Generic/Check.png</normaloff>:/Resource/Generic/Check.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Paused</string>
        </property>
        <property name="statusTip">
         <string notr="true">Paused</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/Resource/Media/PauseMedia.png</normaloff>:/Resource/Media/PauseMedia.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Active</string>
        </property>
        <property name="statusTip">
         <string notr="true">Active</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/Resource/Media/PlayMedia.png</normaloff>:/Resource/Media/PlayMedia.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inactive</string>
        </property>
        <property name="statusTip">
         <string notr="true">Inactive</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/Resource/Media/StopMedia.png</normaloff>:/Resource/Media/StopMedia.png</iconset>
        </property>
       </item>
      </item>
      <item>
       <property name="text">
        <string>Uploads</string>
       </property>
       <property name="statusTip">
        <string notr="true">Uploads</string>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/Resource/Generic/Upload.png</normaloff>:/Resource/Generic/Upload.png</iconset>
       </property>
       <item>
        <property name="text">
         <string>Completed</string>
        </property>
        <property name="statusTip">
         <string notr="true">CompletedUploads</string>
        </property>
        <property name="icon">
         <iconset resource="../Resource.qrc">
          <normaloff>:/Resource/Generic/Check.png</normaloff>:/Resource/Generic/Check.png</iconset>
        </property>
       </item>
      </item>
     </widget>
     <widget class="QStackedWidget" name="stackedWidgetTransfers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageDownloads">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitterDownloads">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frameDownloads">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>1</verstretch>
            </sizepolicy>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayoutDownloads">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
          <widget class="QTabWidget" name="tabWidgetDownloadDetails">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>3</number>
           </property>
           <widget class="QWidget" name="tabGeneral">
            <attribute name="icon">
             <iconset resource="../Resource.qrc">
              <normaloff>:/Resource/Generic/Properties.png</normaloff>:/Resource/Generic/Properties.png</iconset>
            </attribute>
            <attribute name="title">
             <string>General</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>-289</y>
                  <width>377</width>
                  <height>375</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Transfer</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="0">
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>Uploaded:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>Downloaded:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QLabel" name="label_3">
                      <property name="text">
                       <string>Wasted:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="1">
                     <widget class="QLabel" name="label_4">
                      <property name="text">
                       <string>Up Limit:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QLabel" name="label_5">
                      <property name="text">
                       <string>Down Limit:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>Time Active:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="2">
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string>Share Ratio:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2">
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>Connections:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="2">
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>Reannounce In:</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_2">
                   <property name="title">
                    <string>Information</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_2">
                    <property name="leftMargin">
                     <number>2</number>
                    </property>
                    <property name="topMargin">
                     <number>4</number>
                    </property>
                    <property name="rightMargin">
                     <number>2</number>
                    </property>
                    <property name="bottomMargin">
                     <number>2</number>
                    </property>
                    <property name="spacing">
                     <number>2</number>
                    </property>
                    <item row="1" column="1">
                     <widget class="QToolButton" name="toolButton">
                      <property name="text">
                       <string>Browse</string>
                      </property>
                      <property name="icon">
                       <iconset resource="../Resource.qrc">
                        <normaloff>:/Resource/Library/BrowseFolder.png</normaloff>:/Resource/Library/BrowseFolder.png</iconset>
                      </property>
                      <property name="iconSize">
                       <size>
                        <width>24</width>
                        <height>24</height>
                       </size>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="2" colspan="2">
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>Save Path:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1" colspan="3">
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>Created On:</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QLabel" name="label_12">
                      <property name="text">
                       <string>Comment</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="2" colspan="2">
                     <widget class="QPlainTextEdit" name="plainTextEdit"/>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGroupBox" name="groupBox_3">
                   <property name="title">
                    <string>Hashes</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_14">
                      <property name="text">
                       <string>SHA1: </string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_15">
                      <property name="text">
                       <string>Tiger-Root:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_16">
                      <property name="text">
                       <string>ED2K:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_17">
                      <property name="text">
                       <string>MD5:</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_13">
                      <property name="text">
                       <string>BitTorrent:</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabTrackers">
            <attribute name="icon">
             <iconset resource="../Resource.qrc">
              <normaloff>:/Resource/Network/Discovery.png</normaloff>:/Resource/Network/Discovery.png</iconset>
            </attribute>
            <attribute name="title">
             <string>Trackers</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableView" name="tableViewTrackers"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabPeers">
            <attribute name="icon">
             <iconset resource="../Resource.qrc">
              <normaloff>:/Resource/Chat/Friends.png</normaloff>:/Resource/Chat/Friends.png</iconset>
            </attribute>
            <attribute name="title">
             <string>Peers</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableView" name="tableViewPeers"/>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabFiles">
            <attribute name="icon">
             <iconset resource="../Resource.qrc">
              <normaloff>:/Resource/Library/OpenDownloadFolder.png</normaloff>:/Resource/Library/OpenDownloadFolder.png</iconset>
            </attribute>
            <attribute name="title">
             <string>Files</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTreeView" name="treeView"/>
             </item>
            </layout>
            <zorder>treeWidgetTransfers</zorder>
            <zorder>treeView</zorder>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageUploads">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitterUploads">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QFrame" name="frameUploads">
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayoutUploads">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
